bikespace_dashboard/js/components/map.js

```diff
@@ -1,140 +1,141 @@
+      minutes: 'for <strong>less than an hour</strong>',
+      hours: 'for <strong>several hours</strong>',
+      overnight: '<strong>overnight</strong>',
+      multiday: 'for <strong>several days</strong>',
     };
-    let issue_tags = {};
-    for (let entry of Object.values(ia)) {
+    const issue_tags = {};
+    for (const entry of Object.values(ia)) {
       issue_tags[entry.id] = [
         `<div class="issue issue-${entry.id.replace('_', '-')}" `,
-          `style="border-color:${entry.color};`,
-            `background-color:${entry.color_light};"`,
-        `>${entry.label_long}</div>`
-      ].join("");
+        `style="border-color:${entry.color};`,
+        `background-color:${entry.color_light};"`,
+        `>${entry.label_long}</div>`,
+      ].join('');
     }
 
     // use templates to generate popup content
-    for (let point of this.shared_state.display_data) {
+    for (const point of this.shared_state.display_data) {
       // display issue chips in priority order
       point.issues.sort(
         (a, b) => ia[a].render_priority - ia[b].render_priority
-        );
-      let issues = point.issues.map(
-        (x) => issue_tags[x] ?? `<div class="issue">${x}</div>`
-        ).join(" ");
-      let parking_time = new Date(point.parking_time);
-      let parking_time_desc = parking_time.toLocaleString("en-CA", {
-        "dateStyle": "full",
-        "timeStyle": "short"
+      );
+      const issues = point.issues
+        .map(x => issue_tags[x] ?? `<div class="issue">${x}</div>`)
+        .join(' ');
+      const parking_time = new Date(point.parking_time);
+      const parking_time_desc = parking_time.toLocaleString('en-CA', {
+        dateStyle: 'full',
+        timeStyle: 'short',
       });
-      let comments = `<strong>Comments:</strong> ${
-        point.comments ? point.comments : "<em>none</em>"
+      const comments = `<strong>Comments:</strong> ${
+        point.comments ? point.comments : '<em>none</em>'
       }`;
 
-      let content = [
-        `<div class="issue-list">`,
-        `<strong>Issues:</strong> ${issues ? issues : "<em>none</em>"}`,
-        `</div>`,
+      const content = [
+        '<div class="issue-list">',
+        `<strong>Issues:</strong> ${issues ? issues : '<em>none</em>'}`,
+        '</div>',
         `<p>This person wanted to park ${
-          duration_descr[point.parking_duration] ?? 
+          duration_descr[point.parking_duration] ??
           `<strong>${point.parking_duration}</strong>`
         } on <strong>${parking_time_desc}</strong></p>`,
         `<p>${comments}</p>`,
-        `<p class="submission-id">ID: ${point.id}</p>`
-      ].join("");
+        `<p class="submission-id">ID: ${point.id}</p>`,
+      ].join('');
 
       // BUILD MARKERS
       // set up custom markers
-      let BaseIcon = L.Icon.extend({
+      const BaseIcon = L.Icon.extend({
         options: {
-          shadowUrl: "./leaflet/images/marker-shadow.png",
-          iconSize:     [36, 36],
```

Comment:
I actually prefer the aligned brackets here

Reply:
sorry, must have removed the line during a merge.

Reply:
let's see what I can do with prettier...

Reply:
prettier's devs explicitly stated they [would not support vertical alignment](https://github.com/prettier/prettier/issues/188#issuecomment-272627905) and it'd be too much effort to use either eslint's [vertical alignment plugin](https://www.npmjs.com/package/eslint-plugin-align-assignments) along with prettier.

would therefore leave it at this